### PR TITLE
Fields on the Frontend and Admin sign in pages lose focus #141. Fixed…

### PR DESCRIPTION
**Pull request by askrht**
… and tested.

Fixes https://github.com/marmelab/javascript-boilerplate/issues/141

### CODE REVIEW

File: src/admin/js/user/SignIn.js [hunk 1]
```diff
@@ -27,15 +27,14 @@ const signInSchema = buildSchema({
...
-        <TextField
-            errorText={touched && error}
-            {...{ floatingLabelText, type, hintText }}
-            {...inputProps}
-            {...props}
-            fullWidth
-        />;
+const renderInput = (field) => {
+    const { meta: { touched, error } = {}, input: { ...inputProps }, ...props } = field;
+    return(<TextField
```
Comment (djhi) [inline src/admin/js/user/SignIn.js]: Please apply the same style as us:
```js
const renderInput = ({ meta: { touched, error } = {}, input: { ...inputProps }, ...props }) =>
    <TextField
        errorText={touched && error}
        {...inputProps}
        {...props}
        fullWidth
    />;
```

Comment (djhi) [inline src/admin/js/user/SignIn.js]: Btw, `input: { ...inputProps }` should be `input: inputProps` if you just want to rename it

**Comment by askrht** [comment]
@djhi I have restyled the renderInput method for `Admin` UI but I did not change the way inputProps are de-structured because that breaks the sign-in flow for `Admin` UI. It is a good idea to add a UI test to guard against this happening accidentally.

**Comment by djhi** [comment]
Thanks a lot @askrht !

**Pull request merged**